docs/DafnyRef/Statements.md

```diff
@@ -296,13 +334,10 @@ as the number of named out-parameters. These expressions are
 evaluated, then they are assigned to the out-parameters, and then the
 method terminates.
 
-## 20.5. Yield Statement {#sec-yield-statement}
-````grammar
-YieldStmt = "yield" [ Rhs { "," Rhs } ] ";"
-````
+## 20.5. Yield Statement ([grammar](#g-yield-statement)) {#sec-yield-statement}
 
 A yield statement can only be used in an iterator.
-See [Section 16](#sec-iterator-types) for more details
+See [Section 15](#sec-iterator-types) for more details
```

Comment:
```suggestion
See [iterator types](#sec-iterator-types) for more details
```
Depending on a section number is irrelevant if the document has clickable hyperrefs.

Reply:
But for those who read a paper copy they are helpful!

Reply:
Facts:
* I think 99.99% of people will use a clickable version (the fact is that I believe this)
* Manually writing section numbers is our maintenance burden that even LaTeX users don't have.
* For those to dare to print this reference guide on not less than 190 sheets of paper which is roughly two pounds of paper if you print two-sided, they will have the problem that it's always outdated. And if they print less, then these links don't serve any purpose.